client/components/App.js

```diff
@@ -1,36 +1,55 @@
 
 import React from "react";
 import ProductDetail from './ProductDetail/ProductDetail';
+import RelatedProductsAndOutfits from './RelatedPoductsAndOutfit.jsx'
 import axios from 'axios';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      currentProduct: {}
+      allProducts: [],
+      currentProduct: {},
+      currentProductId: null
     }
   }
 
   componentDidMount() {
     let productId = '37313';
     axios.get(`/api/products/${productId}`).then(({ data }) => {
       this.setState({
-        currentProduct: data
+        currentProduct: data,
+        currentProductId: data.id
+      })
+    })
+    axios.get('/api/products').then(({data}) => {
+      this.setState({
+        products: data
       })
     })
   }
+
+  setProduct(req) {
+    // well.. this will be more of an event handler. Can pinpoint to a an aleady existing product from our state, rather than continuously 'getting'
+  }
+
   render() {
+    //console.log(this.state)
     const { name } = this.props;
     const { currentProduct } = this.state;
+    const {allProducts} = this.state;
```

Comment:
can combine line 40 with line 39: const { currentProduct, allProducts } = this.state;